x-pack/legacy/plugins/monitoring/public/components/chart/_chart.scss

```diff
@@ -68,3 +68,16 @@
   white-space: nowrap;
   margin-left: $euiSizeXS;
 }
+
+.monChart__tooltipLabel,
+.monChart__tooltipValue {
+  text-align: left;
+  font-size: $euiFontSizeXS;
+  padding: $euiSizeXS;
+  word-wrap: break-word;
+  white-space: normal;
+}
+
+.monChart__tooltipLabel {
+  font-weight: $euiFontWeightBold;
+}
```

Comment:
These look like additions. Why were they necessary?

Reply:
One of the `scss` file that was bundled with the dead code still had classes that were used in a different place: https://github.com/igoristic/kibana/blob/7ac8e4d9ccc2a9e064d8a5621cd62453d1006164/x-pack/legacy/plugins/monitoring/public/components/chart/info_tooltip.js So, I moved them to a more relevant place

Reply:
Gotcha, makes sense, thanks!